src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -134,24 +134,24 @@ export default {
 .messages {
 	flex: auto;
 	display: flex;
-	padding: 8px 0 8px 0;
+	padding: var(--default-grid-baseline) 0;
 	flex-direction: column;
 	width: 100%;
 	min-width: 0;
 
 	&__avatar {
 		position: sticky;
 		top: 0;
-		height: 52px;
-		width: 52px;
-		padding: 18px 10px 10px 10px;
+		padding-inline: var(--chat-spacing, 8px);
```

Comment:
IMO it would be better if it's 10px at least. It looks really close to the edge in main view.

Reply:
Agree, also looks too close for me in the main window on a small screen.

Maybe even 12px to make it on the grid

Reply:
keep it as 8px, so 8+32+8 for the complete width. Does 2px to the edge reaaly make that difference?